src/Lykke.Service.Tier.AzureRepositories/TierUpgradeRequestsRepository.cs

```diff
@@ -29,11 +39,31 @@ public async Task<ITierUpgradeRequest> GetAsync(string clientId, AccountTier tie
                 TierUpgradeRequestEntity.GenerateRk(clientId));
         }
 
+        public async Task<IReadOnlyList<ITierUpgradeRequest>> GetByClientAsync(string clientId)
+        {
+            return (await _tableStorage.GetDataAsync(new[]
+            {
+                new Tuple<string, string>(TierUpgradeRequestEntity.GeneratePk(AccountTier.Apprentice), clientId),
+                new Tuple<string, string>(TierUpgradeRequestEntity.GeneratePk(AccountTier.Advanced), clientId),
+                new Tuple<string, string>(TierUpgradeRequestEntity.GeneratePk(AccountTier.ProIndividual), clientId)
+            })).ToList();
+        }
+
         public async Task<IReadOnlyList<ITierUpgradeRequest>> GetByTierAsync(AccountTier tier)
         {
             return (await _tableStorage.GetDataAsync(TierUpgradeRequestEntity.GeneratePk(tier))).ToList();
         }
 
+        public async Task<IReadOnlyList<ITierUpgradeRequest>> GetAllAsync()
```

Comment:
Don't we need pagination here? Can we expect or even guarantee low amount of requests?